### PR TITLE
Pls merge me I'm a crucial feature

### PR DESCRIPTION
**Pull request by sue-m-oh**
A very descriptive reasoning

### CODE REVIEW

File: hello_mina.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("mina says hi")
```
Comment (shenanigansd) [inline hello_mina.py]:
```suggestion
print("Mina says hi")
```
Proper nouns should be capitalized

**Review by shenanigansd — CHANGES_REQUESTED** [review]
Almost there!

File: hello_mina.py (hunk 1 above)
Comment (D0rs4n) [reply hello_mina.py]: discord

**Review by lemonyte — CHANGES_REQUESTED** [review]
Punctuation would be a nice addition, other than that looks fine.

File: hello_mina.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("Mina says hi")
```
Comment (lemonyte) [inline hello_mina.py:1]:
```suggestion
print('Mina says "Hi".')
```

**Pull request merged**